third_party/move/move-compiler-v2/tests/checking/receiver/call_errors.exp

```diff
@@ -0,0 +1,43 @@
+
+Diagnostics:
+error: undeclared receiver function `receiver_wrongly_spelled` for type `S<u64>`
+   ┌─ tests/checking/receiver/call_errors.move:16:9
+   │
+16 │         s.receiver_wrongly_spelled(x);
+   │         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+error: expected 2 items but found 3
```

Comment:
nit: while this is technically correct, we may want specialize an error message for receiver style calls (something like: expected 2 arguments for the receiver function, but found 1). On a quick parse of the error message, the user may think the compiler is wrong.